Makefile

```diff
@@ -0,0 +1,34 @@
+COMPILESVG=inkscape
+
+PDFLATEXFLAGS=-halt-on-error -interaction nonstopmode
+
+all: form.pdf
+
+clean:
+	rm -f *.aux *.lof *.log *.lot *.fls *.out *.toc *.fmt *.fot *.cb *.cb2 *.lol *.pdf
+
+
+form.pdf: form.tex fig-SourceBots.pdf
+	pdflatex $(PDFLATEXFLAGS) $<
+
+fig-%.pdf: fig-%.svg
+ifeq ($(COMPILESVG),inkscape)
+	inkscape -A `pwd`/$@ `pwd`/$<
+else
+ifeq ($(COMPILESVG),svg2pdf)
+	svg2pdf $< $@
+else
+	echo "Unknown COMPILESVG."; false
+endif
+endif
+
+watch:
+	make
+	while true; do \
+		inotifywait --event CLOSE_WRITE -q -r *.tex; \
+			make; \
+		make; \
```

Comment:
Why is there a double `make` here? (Also why is one of them extra indented?)

Reply:
because when I was fixing the previous error I was being an idiot. fixed now. 48d519d